db/attachment_test.go

```diff
@@ -1548,3 +1549,34 @@ func TestGetAttVersion(t *testing.T) {
 		})
 	}
 }
+
+func TestLargeAttachments(t *testing.T) {
+	context, err := NewDatabaseContext("db", base.GetTestBucket(t), false, DatabaseContextOptions{})
+	assert.NoError(t, err, "Couldn't create context for database 'db'")
+	defer context.Close()
+	db, err := CreateDatabase(context)
+	assert.NoError(t, err, "Couldn't create database 'db'")
+
+	normalAttachment := make([]byte, 10*1024*1024)
+	hugeAttachment := make([]byte, 30*1024*1024)
+	_, _ = rand.Read(normalAttachment)
+	_, _ = rand.Read(hugeAttachment)
```

Comment:
Could we have sizes that don't border a size limit (to avoid unintentionally hitting any off-by-one / `<` vs `<=` cases along various parts of the stack?)

e.g:
- 15MB // ok
- 25MB // covers old CB doc too large case
- 35MB // covers the EOF case